Update encryption table (#157)
decentralized-identity/decentralized-web-node: spec/spec.md

A conforming implementation ****MUST**** be capable of encrypting and decrypting data stored in Decentralized Web Nodes 
using the following combinations of cryptographic schemes. Each scheme is a pair, wherein the first item is the asymmetric 

> encryption scheme used to encrypt the symmetric key it is paired with, and the symmetric key is used to encrypt the actual 
> data being protected.

Reviewer: It makes no sense to "encrypt the symmetric key" nor any other key.  I *think* the following rephrase more accurately communicates the intended meaning.
```suggestion
encryption scheme used to encrypt the data, and the second is the symmetric key used with that scheme to encrypt the actual 
data being protected.
```

Reviewer 2: In the first case, it makes sense to me. You can encrypt keys too. It's just plaintext input to the encryption fn that happens to also be a key to another fn. this is not uncommon

Author: @TallTed the symmetric keys are what you use to encrypt the actual data, and because the symmetric keys need to be shared with recipients, you encrypt the symmetric key using each recipient's asymmetric public key.

Reviewer: @csuwildcat -- OK... I think I understand what is meant to be communicated now. I'll take another whack at the wording shortly.
